From 821a715f75d4659ae98f4e2d0fcd4ecf660c8a2b Mon Sep 17 00:00:00 2001
From: Vassil Vassilev <vvasilev@cern.ch>
Date: Fri, 28 Mar 2014 10:48:12 +0100
Subject: [PATCH 14/57] Access sema internals and revert the list of unused
 decls.

---
 clang/include/clang/Sema/Sema.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/clang/include/clang/Sema/Sema.h b/clang/include/clang/Sema/Sema.h
index d8b2546b81a3..947cf1fc695d 100644
--- a/clang/include/clang/Sema/Sema.h
+++ b/clang/include/clang/Sema/Sema.h
@@ -12798,6 +12798,7 @@ protected:
   friend class ASTReader;
   friend class ASTDeclReader;
   friend class ASTWriter;
+  friend class DeclUnloader;
 
 public:
   /// Retrieve the keyword associated
-- 
2.34.1

